### PR TITLE
Create ObscureXwing.yaml

### PR DESCRIPTION
**Pull request by ObscureXwing**

### CODE REVIEW

File: _data/signed/ObscureXwing [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name : Simon Bright ( I run a technical youtube channel in Russian, I help people solve their problems and learn Linux! Also in my free time I compile kernels for Debian 
+with support for F-Sync and Futex2 patchsets, with support for Ashmem and Binder for Anbox, so that people can easily get the opportunity to play on GNU/Linux 
+and at the same time not lose the ability to work! https://www.youtube.com/channel/UCnQfpqJI9wjcqW-4JS8k-Rg/ )
+name : Simon Bright
```
Comment (gorshunovr) [inline _data/signed/ObscureXwing:4]: Please, just leave lines 4 and 5.

**Review by gorshunovr — COMMENTED** [review]
Commented.

File: _data/signed/ObscureXwing (hunk 1 above)
Comment (ObscureXwing) [reply _data/signed/ObscureXwing]: Thanks for the help ! I hope now my signature fits the format?

**Comment by gorshunovr** [comment]
Sorry, Simon. It needs to be with 2 lines only. Like this:
```
name : Simon Bright (F-Sync and Futex2 Debian kernels, with support for Ashmem and Binder for Anbox)
link : https://www.youtube.com/channel/UCnQfpqJI9wjcqW-4JS8k-Rg/
```

**Comment by ObscureXwing** [comment]
> Sorry, Simon. It needs to be with 2 lines only. Like this:
> 
> ```
> name : Simon Bright (F-Sync and Futex2 Debian kernels, with support for Ashmem and Binder for Anbox)
> link : https://www.youtube.com/channel/UCnQfpqJI9wjcqW-4JS8k-Rg/
> ```
Ok, no problem, i edited the file! Sorry for delay

**Pull request merged**